docker/docker-compose-mini.yml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
+
+  mongo: # 27017
+    image: mongo:latest
+    network_mode: "host"
+    privileged: true
```

Comment:
As a rule of thumb, anything that a node tries to connect to while PXE booting needs to be on the host network (privileged).

* rabbitmq
* dhcp
* dhcp-proxy
* http (south-bound)
* syslog
* statsd (not sure)
* tftp